Apto/Plugins/PartsListElement/Infrastructure/PartsListElementBundle/PartsListElementBundle.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Apto\Plugins\PartsListElement\Infrastructure\PartsListElementBundle;
+
+use Apto\Base\Infrastructure\AptoBaseBundle\AbstractAptoBundle;
+use Apto\Plugins\CustomText\Infrastructure\CustomTextBundle\DependencyInjection\CustomTextExtension;
```

Comment:
refers to the wrong extension, should be PartsListElementExtension not CustomTextExtension